.github/workflows/markdown_linter.yml

```diff
@@ -10,17 +10,31 @@ on:
 jobs:
   markdown-link-check:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x]
     steps:
     - uses: actions/checkout@master
-    - uses: gaurav-nelson/github-action-markdown-link-check@v1
```

Comment:
why did you remove this? Copy-past mistake?

Reply:
No, trying to use a different linter. I might not be doing this correctly, however. Mentioned in the comment above, probably need your help on this.

Reply:
Reverted.

Reply:
I just realized that I was misreading the job name -- I thought it said `markdown-lint-check`, not `markdown-link-check`!

Reply:
@kevmoo PTAL again. Fixed.